### PR TITLE
Add new "Tiny" screen size for Adafruit 2.8 screen

### PR DESCRIPTION
**Pull request by Kerdic**
1. Add new "tiny" screen size for Adafruit 2.8 screen.
2. Include logic to show PADD version in orange if the process fails to retrieve the latest version.
3. Update the version log file to include the current version and the retrieved latest version.

### CODE REVIEW

**Comment by dschaper** [comment]
What are the display dimensions?

`tput cols`
`tput lines`

**Comment by Kerdic** [comment]
So, it should show this tiny screen if the columns are >= 50 and <= 59 or if the lines = 21.

Is that what you are asking for?

**Comment by Kerdic** [comment]
Nevermind, let me figure out how to run those commands on that screen, and I will get the dimensions.

**Comment by dschaper** [comment]
Yeah, just need to be able to set up a terminal window with the same rows/columns size to check it all out.

**Comment by Kerdic** [comment]
ok, I went a bit different route, and just had the script output the $console_width and $console_height variables to the piHoleVersion file.

width=53
height=20

And, I misread my code change earlier, so it runs the tiny sections if the lines are less than 21, not equal to 21.

So, maybe I should change the mini from -lt 50 to be instead -lt 53? Since the Tiny screen uses 53 length of character.

File: padd.sh [hunk 1]
```diff
@@ -588,6 +623,9 @@ PrintLogo() {
   elif [ "$1" = "mini" ]; then
     CleanEcho "${padd_text}${dim_text}mini${reset_text}  ${mini_status_}"
     CleanEcho ""
+  elif [ "$1" = "tiny" ]; then
+    CleanEcho "${bold_text} ${reset_text}${padd_text}${dim_text}tiny ${reset_text}Pi-hole® ${core_version_heatmap}v${core_version}${reset_text}, Web ${web_version_heatmap}v${web_version}${reset_text}, FTL ${ftl_version_heatmap}v${ftl_version}${reset_text}"
+    CleanPrintf "          PADD ${padd_version_heatmap}${padd_version}${reset_text} ${tiny_status_}${reset_text}\e[0K\\n"
```
Comment (christianboyle) [inline padd.sh:628]: If you add a space before `Pi-hole®` and a space before `PADD` you will get a more aesthetically pleasing alignment: 
zz

Thanks for your patch btw!

Comment (dschaper) [reply padd.sh:628]: Let me know if you want to add this in as a change, I'm good to merge this PR in and get things out. I haven't checked it fully yet, but I think we can get it deployed and make changes if needed later.

Comment (dschaper) [reply padd.sh:628]: @christianboyle I can merge this PR as it is and get the existing code out there. Would you open a PR with the improvements after this is on the master branch?

Comment (christianboyle) [reply padd.sh:628]: @dschaper Sure, that sounds good.

**Comment by dschaper** [comment]
@Kerdic Shoot me an email at dan.schaper@pi-hole.net and I'll get something out to say "Thanks!"

**Pull request merged**